apps/api/src/prisma/prisma.repository.ts

```diff
@@ -142,6 +142,7 @@ export class PrismaRepository {
     expiresAfter: number
   ): Promise<Otp> {
     const timeNow = new Date()
+    await this.invalidateOldOtps(email)
```

Comment:
Please move this to a cron job. Create a cron function in the auth service itself. Ref: https://docs.nestjs.com/techniques/task-scheduling. I would like the old OTPs to be deleted every hour. I believe there is a thing called `CronInterval.HOUR` or something similar in the reference I shared.